docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -1,28 +1,30 @@
 ---
 title: Create an ASP.NET Office Add-in that uses single sign-on
```

Comment:
For SEO, consider adding "(SSO)" to title since people use it in Search.

Reply:
Added.